SU-SWS/saa_victoria_subtheme
# ADAPT-696: Added min-height to desktop masthead.

# READY FOR REVIEW

# Summary
- Add a min-height for large screen and up on the masthead, to fix search bar position now that line height doesn't keep the height (When no menu is there)

# Review By (Date)
- 9/15 asap

# Urgency
- medium, visible to clients right now here: https://saayoungalumni.sites.stanford.edu/ (search doesn't have space below because masthead site name line height was changed, i believe.

# Steps to Test

1. Pull down the branch if your local is up to date after yesterday's release, or look at: https://saayoungalumni.sites.stanford.edu/  (I put this branch onto that site, so you can see the code fixing it there)
2. Clear the site cache
3. See that the su-masthead has a min-height of 96px (right now this is hard-coded to do a quick fix. We can also discuss how we want to do this / handle this differently, but we want to at least fix this TODAY since clients /stakeholders are looking)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-696
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
